nestkernel/connection_manager.cpp

```diff
@@ -1218,188 +1218,202 @@ nest::ConnectionManager::split_to_neuron_device_vectors_( const size_t tid,
 }
 
 void
-nest::ConnectionManager::get_connections( std::deque< ConnectionID >& connectome,
+nest::ConnectionManager::get_connections_( const size_t tid,
+  std::deque< ConnectionID >& connectome,
   NodeCollectionPTR source,
```

Comment:
Leaving out the unused name should avoid the warnings.
```suggestion
  NodeCollectionPTR,
```